tests/functional/generate_pages.py

```diff
@@ -193,7 +195,7 @@
 """
 
 CALCULATED_SUMMARY_LABEL_GETTER = Template(
-    r"""  ${answerName}Label() { return `[data-qa=${answerId}-label]`; }
+    r"""  ${answerName}Label() { return `#{answerId}-label`; }
```

Comment:
This is now based off a specific id, rather than a data-qa selector (which is what the card specified to move to).

Reply:
See previous comment